CONTRIBUTING.md

```diff
@@ -1,54 +1,39 @@
-<table>
-    <tr>
-        <!-- Do not translate this table -->
-        <td> Read these guidelines in </td>
-        <td><a href="/CONTRIBUTING.md"> English </a></td>
-        <td><a href="/docs/arabic/CONTRIBUTING.md"> عربي </a></td>
-        <td><a href="/docs/chinese/CONTRIBUTING.md"> 中文 </a></td>
-        <td><a href="/docs/portuguese/CONTRIBUTING.md"> Português </a></td>
-        <td><a href="/docs/russian/CONTRIBUTING.md"> Русский </a></td>
-        <td><a href="/docs/spanish/CONTRIBUTING.md"> Español </a></td>
-        <td><a href="/docs/greek/CONTRIBUTING.md"> Ελληνικά </a></td>
-    </tr>
-</table>
+<!-- do not translate this -->
+| [Read these guidelines in your spoken language](/docs/i18n-languages) |
```

Comment:
```suggestion
| [Read these guidelines in other languages](/docs/i18n-languages) |
```
If we want to say 'spoken language' then 'own language' is a more standard way to put it, but 'other languages' is probably best because a) there are multiple languages there and b) the language they speak might not be amongst them.